### ISSUE TITLE
WCAG2AA: 4_1.4_1_2.H91.Div.Name - dropdownmenu with <div role="button"> and text '...' is not a discernible text

### ISSUE THREAD

**Issue by JoaoEnesGoncalves** [lumada-design/hv-uikit-react#777]
**Describe the bug**
button with text '...'  return error "Buttons must have discernible text" 

- WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Div.Name
- https://dequeuniversity.com/rules/axe/3.4/button-name?application=axeAPI

**To Reproduce**
https://pentaho.github.io/hv-uikit-react/?path=/story/components--dropdownmenu

1. run
```
node C:\Git\hv-uikit-react\automation\robot\storybook\_resources/accessibility.js --url=http://localhost:9002/iframe.html?id=coredropdownmenu--dropdownmenu1 --standard=WCAG2AA --element="#root>div.Component-content-4" --actions="click element div[class|='DropDownMenu'][role='button']"
```

**Screenshots**
```
{
  documentTitle: 'Storybook',
  pageUrl: 'http://localhost:9002/iframe.html?id=coredropdownmenu--dropdownmenu1',
  issues: [
    {
      code: 'button-name',
      type: 'error',
      typeCode: 1,
      message: 'Buttons must have discernible text (https://dequeuniversity.com/rules/axe/3.4/button-name?application=axeAPI)',
      context: '<div role="button" tabindex="0" class="DropDownMenu-icon-148 DropDownMenu-iconSelected-149"><div class="MoreOptionsVertical...</div>',
      selector: '#root > div:nth-child(2) > div > div > div > div > div:nth-child(1)',
      runner: 'axe',
      runnerExtras: [Object]
    },
    {
      code: 'WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Div.Name',
      type: 'error',
      typeCode: 1,
      message: 'This element has role of "button" but does not have a name available to an accessibility API. Valid names are: element content, aria-label undefined, aria-labelledby undefined.',
      context: '<div role="button" tabindex="0" class="DropDownMenu-icon-148 DropDownMenu-iconSelected-149"><div class="MoreOptionsVertical...</div>',
      selector: '#root > div:nth-child(2) > div > div > div > div > div:nth-child(1)',
      runner: 'htmlcs',
      runnerExtras: {}
    }
  ]
}
```
